tests/pytests/test_lro_drivers.py

```diff
@@ -46,12 +46,6 @@ def test_spacecraft_name(driver):
 def test_sensor_model_version(driver):
     assert driver.sensor_model_version == 2
 
-def test_detector_start_sample(driver):
-    assert driver.detector_start_sample == 1
-
-def test_detector_start_line(driver):
-    assert driver.detector_start_sample == 1
```

Comment:
Why remove the test instead of updating the value?

Reply:
Because this is no longer defined in the concrete driver, so this is just testing the base Driver class.

Reply:
I thought about adding a test for the base Driver class, but I don't think it's worth it